src/app/pages/maps/search-map/entity/Location.ts

```diff
@@ -0,0 +1,4 @@
+export class Location {
+  constructor(public latitude: number = 54, public longitude: number = 33) {
```

Comment:
This doesn't look right as basically your location entity always has some specific location. You better initialize it where you use it.